zerver/tests/webhooks/test_gosquared.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from zerver.lib.test_classes import WebhookTestCase
+
+class GoSquaredHookTests(WebhookTestCase):
+    STREAM_NAME = 'gosquared'
+    URL_TEMPLATE = "/api/v1/external/gosquared?stream={stream}&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'gosquared'
+
+    # Note: Include a test function per each distinct message condition your integration supports
+    def test_traffic_message(self):
+        # type: () -> None
+        expected_subject = u"GoSquared"
+        expected_message = u"requestb.in has 0 visitors online."
```

Comment:
Can we hand-edit our test data so that this test is a little more meaningful?  All the numeric values in the test data are zero, so this test could pass even if we're pulling from the wrong field.